CONTRIBUTING.md

```diff
@@ -84,6 +84,21 @@ Some of the packages within jest have a `__tests__` directory. This is where uni
 
 There will be situations however where the work you have done cannot be tested alone using unit tests. In situations like this, you should write an integration test for your code. The integration tests reside within the `e2e` directory. Within this directory, there is a `__tests__` directory. This is where you will write the integration test itself. The tests within this directory execute jest itself using `runJest.js` and assertions are usually made on one if not all the output of the following `status`, `stdout` and `stderr`. The other sub directories within the `e2e` directory are where you will write the files that jest will run for your integration tests. Feel free to take a look at any of the tests in the `__tests__` directory within `e2e` to have a better sense of how it is currently being done.
 
+It is possible to run the integration test itself manually to inspect that the new behaviour is indeed correct. Here is a small code snippet of how to do just that. This is useful when debugging a failing test.
+
+```bash
+$ cd e2e/clear-cache
+$ ../../packages/jest-cli/bin/jest.js
```

Comment:
@SimenB I couldn't achieve your behavior by running ../../jest within the folder. Possibly an error on my machine? Anyways doesn't ../../jest just invoke ../../packages/jest-cli/bin/jest.js?

Reply:
yeah, that's fine. Maybe do `node  ../../packages/jest-cli/bin/jest.js` so it's easier for people to understand they can do `node --inspect` or use `ndb`